src/components/uikit/button/index.tsx

```diff
@@ -0,0 +1,14 @@
+import Link from "next/link";
+import React from "react";
+
+interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  pathName: string;
+}
+
+export default function LinkButton(props: ButtonProps) {
+  return (
+    <Link href={props.pathName} className="btn-nav">
+      {props.title}
+    </Link>
+  );
+}
```

Comment:
```suggestion
import { LinkProps } from 'next/link'

interface LinkButtonProps extends LinkProps {}

export default function LinkButton(props: LinkButtonProps) {
  return (
    <Link {...props} href={props.href} className={clsx('btn-nav', props.className)}>
      {props.children}
    </Link>
  );
}
```

As you use `Link` component here, you should extend `LinkProps` type. Also, you should pass all `props` to a Link component using `{...props}` construction (it'll allow you to use other props like: `rel`, `target`, etc).
And, please, use `children` for nested element props